inc/REST/AuthController.php

```diff
@@ -2,8 +2,6 @@
 
 namespace RemoteDataBlocks\REST;
 
-use WP_REST_Controller;
```

Comment:
Is this a code style thing or actually enforced by Psalm? I sort of prefer the use statements for consistency, but don't know how others feel

Reply:
It probably can be, but, It's not enforced anywhere. I have mixed feelings myself. On one hand, I aesthetically dislike the `\` but OTOH I feel like it communicates more clearly in down page context.

I'll put it back.